packages/auth/src/mfa/assertions/totp.test.ts

```diff
@@ -180,7 +185,9 @@ describe('core/mfa/totp/assertions/TotpMultiFactorAssertionImpl', () => {
           sessionInfo: 'verification-id'
         }
       });
-      expect(session.auth).to.eql(auth);
+      if (session.user) {
```

Comment:
I think we should expect `session.user` is not null here and elsewhere.

Reply:
+1, we should check that it's not null and we can also `expect(session.user).to-eql(user)` once user is initialized in beforeEach.

We should also, as a result remove the `if (session.user)` check.